.github/workflows/container-tests.yml

```diff
@@ -3,7 +3,56 @@ on:
     types:
       - created
 jobs:
+  distgen-check:
```

Comment:
Why not separate GitHub Action?

Reply:
Because the is no possibility currently to make multiple Github Actions depend on each other.
And we want to run container tests only when the distgen check passes.

Reply:
But I could make a composite GHA from this, if this works.